### PR TITLE
add instructions for set up

### PR DESCRIPTION
**Pull request by njogz**
This adds some instructions on how to set up the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+CHT Pipeline
+========================
+
+CHT Pipeline is a tool used to define data models for transforming the raw data we get from Couch DB into models that can then be queried to build dashboards
+
+# Installation
```
Comment (garethbowen) [inline README.md:6]: Let's be more opinionated here. If these are equivalent then pick the best/easiest option. If they're not equivalent then let's document why you should pick one over the other.

Also, is this dev environment or production installation? Let's be clear.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+This project canbe run in one of two ways:
```
Comment (garethbowen) [inline README.md:7]:
```suggestion
This project can be run in one of two ways:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+1. Using the docker image and passed as an argument to [cht-sync](https://github.com/medic/cht-sync)
+2. Using the local installation of DBT
+
+## Docker
+1. Follow the instructions in [cht-sync](https://github.com/medic/cht-sync) to set up the required environment variables
+1. Add the path to your pipline banch to the [docker-compose.postgres.yml](https://github.com/medic/cht-sync/blob/main/docker-compose.postgres.yml#L13) file in the cht-sync project
```
Comment (garethbowen) [inline README.md:13]: Change this to be an environment variable. If you edit the docker-compose source directly then checking out later versions may conflict or you might accidentally commit it back to the main branch.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+1. Run docker-compose up to run the pipeline
```
Comment (garethbowen) [inline README.md:14]:
```suggestion
1. Run `docker-compose up` to run the pipeline
```

File: README.md (hunk 3 above)
Comment (garethbowen) [inline README.md:13]:
```suggestion
1. Add the path to your pipeline branch to the [docker-compose.postgres.yml](https://github.com/medic/cht-sync/blob/main/docker-compose.postgres.yml#L13) file in the cht-sync project
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Thanks! Suggestions inline.

File: README.md [hunk 5]
```diff
@@ -3,17 +3,7 @@ CHT Pipeline
...
-# Installation
-This project can be run in one of two ways:
-1. Using the docker image and passed as an argument to [cht-sync](https://github.com/medic/cht-sync)
-2. Using the local installation of DBT
+# Dev installation instructions
 
-## Docker
 1. Follow the instructions in [cht-sync](https://github.com/medic/cht-sync) to set up the required environment variables
-1. Add the path to your pipeline branch to the [docker-compose.postgres.yml](https://github.com/medic/cht-sync/blob/main/docker-compose.postgres.yml#L13) file in the cht-sync project
 1. Run `docker-compose up` to run the pipeline
```
Comment (lorerod) [inline README.md:9]: Let's clarify if I have to run it on the cht-pipeline directory or the cht-sync directory.

**Review by lorerod — COMMENTED** [review]
Thanks, @njogz, for this documentation. I got my instance up and running with the instructions. I didn´t have time to look at the code in detail to know how everything works together. It may be interesting to have instructions on how to see all working together in the future, just a few steps to get a glimpse of what is happening.

**Review by lorerod — APPROVED** [review]
Thanks @njogz

File: README.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+CHT Pipeline
+========================
+
+CHT Pipeline is a tool used to define data models for transforming the raw data we get from Couch DB into models that can then be queried to build dashboards
+
+# Dev installation instructions
+
+1. Follow the instructions in [cht-sync](https://github.com/medic/cht-sync) to set up the required environment variables
+1. Run `docker-compose up` in the cht-sync directory to run the pipeline. Please ensure the `DATAEMON_INITAL_PACKAGE` env variable is set to your preferre branch of the cht-pipline repo.
```
Comment (andrablaj) [inline README.md:9]:
```suggestion
1. Run `docker-compose up` in the cht-sync directory to run the pipeline. Please ensure the `DATAEMON_INITAL_PACKAGE` env variable is set to your preferred branch of the cht-pipeline repo. 
```

**Review by andrablaj — CHANGES_REQUESTED** [review]
Minor typos.

**Review by andrablaj — APPROVED** [review]

**Review by garethbowen — APPROVED** [review]
Ship it!

**Pull request merged**